flink/src/main/java/org/apache/iceberg/flink/source/ScanContext.java

```diff
@@ -61,6 +61,8 @@
   private static final ConfigOption<Long> SPLIT_FILE_OPEN_COST =
       ConfigOptions.key("split-file-open-cost").longType().defaultValue(null);
 
+  private static final ConfigOption<Long> LIMIT = ConfigOptions.key("limit").longType().defaultValue(-1L);
```

Comment:
Do we need this ?  we don't parse the limit from properties, right ?

Reply:
I removed it